src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Card/CardPaymentOperations.cs

```diff
@@ -1,81 +1,84 @@
-    }
+﻿using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CardPaymentOperations : OperationsBase, ICardPaymentOperations
+    {
+        internal CardPaymentOperations(IOperationList operations, HttpClient client)
+        {
+            foreach (var httpOperation in operations)
+            {
+                switch (httpOperation.Rel.Value)
+                {
+                    case PaymentResourceOperations.UpdatePaymentAbort:
+                        Abort = async payload =>
+                            await client.SendAsJsonAsync<ICardPaymentResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.RedirectAuthorization:
+                        RedirectAuthorization = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.ViewAuthorization:
+                        ViewAuthorization = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.DirectAuthorization:
+                        DirectAuthorization = async payload =>
+                            await client.SendAsJsonAsync<CardPaymentAuthorizationResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateCapture:
+                        Capture = async payload =>
+                            await client.SendAsJsonAsync<CaptureResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateCancellation:
+                        Cancel = async payload =>
+                            await client.SendAsJsonAsync<CancellationResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateReversal:
+                        Reverse = async payload =>
+                            await client.SendAsJsonAsync<ReversalResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.RedirectVerification:
+                        RedirectVerification = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.ViewVerification:
+                        ViewVerification = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.DirectVerification:
+                        DirectVerification = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.PaidPayment:
+                        PaidPayment = httpOperation;
+                        break;
+                    default:

```

Comment:
Codacy found an issue: [Remove this empty 'default' clause.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)